frontend/src/store/modules/shoots/getters.js

```diff
@@ -256,7 +256,7 @@ export default {
       const inverse = sortOrder === 'desc' ? -1 : 1
       switch (sortBy) {
         case 'k8sVersion': {
-          items.sort((a, b) => {
+          return [...items].sort((a, b) => {
```

Comment:
was this part of the fix? why is it necessary?

Reply:
Yes, because of

>sortItems did cause mutation error as items are passed from store while generating sorted uids array, so the function must not alter the passed item array

However we should discuss this. maybe there is a better solution

Reply:
This is not the right place to fix this issue. This function should not assume that the items are references to the store state. The clone should be done before calling this function. There we now it comes from the store.

Reply:
moved the copy to function caller